core/src/main/java/org/eclipse/hono/config/ClientConfigProperties.java

```diff
@@ -49,6 +49,22 @@
      * frame during link establishment.
      */
     public static final long DEFAULT_LINK_ESTABLISHMENT_TIMEOUT = 1000L; //ms
+    /**
+     * The default minimum amount of time (milliseconds) to wait before trying to re-establish an
+     * AMQP connection with the peer.
+     */
+    public static final int DEFAULT_RECONNECT_MIN_DELAY = 0; // ms
+    /**
+     * The default maximum amount of time (milliseconds) to wait before trying to re-establish an
+     * AMQP connection with the peer.
+     */
+    public static final int DEFAULT_RECONNECT_MAX_DELAY = 1000; // ms
```

Comment:
I think that this is a little low and may not allow stressed components to catch up again. What about something in the order of 30 seconds instead?

Reply:
30s sounds a bit much too me. I would rather use something like 7s.
That still wouldn't put much pressure on a "stressed component" but would allow for potentially faster recovery if the target component is just being restarted.